### PR TITLE
Make header and notification bar sticky

### LINKED ISSUE #74: Fix (as in keep static) the notification bar on scroll?
Just to open it up to discussion, topic was first mentioned in #72. 

Was also considering whether the top could maybe be fixed? But I'm more in doubt about that, where I think it could feel nice if the notification bar didn't suddenly disappear when you click an option.

### PR DESCRIPTION
**Pull request by emilgoldsmith**
Resolves #74.

Made both header and notification bar sticky. I think it's an all or nothing design wise, because it looks really bad with the scroll bar going over one of the black parts and not the other one. 

I split making the header and notification bar sticky into two commits so one can easily see what it looks like if only the notification bar is sticky UI wise (with a git checkout).

Biggest possible issues with my implementation is:
* It assumes constant pixel size of header and notification bar (Help me front-end developers, is this a correct assumption?)
* Is app-bulk a super bad classname haha? Any better suggestions?

Also, at least on Ubuntu Chrome, I actually think this PR makes the design a lot better for #77. (also note I forked that PR so that commit is contained in this, I just wanted to separate the two as I see them as separate design decisions).
It does this because the forced scrollbar blends in really nicely when it is contained within the grey part of the app and doesn't touch the black parts.

### CODE REVIEW

File: chrome/extension/options.css [hunk 1]
```diff
@@ -24,7 +24,7 @@ body {
   line-height: 1.4em;
   background: #f5f5f5;
   color: #4d4d4d;
-  width: 300px;
+  width: 320px;
```
Comment (martynchamberlin) [inline chrome/extension/options.css]: I'd approve this PR if it weren't for this change and the change in popup.jade. Let's handle this in a separate PR, at e.g. https://github.com/ChessCom/browser-extension/pull/77. 

The migration from 300 to 320 is a little more complicated than just this.

Comment (emilgoldsmith) [inline chrome/extension/options.css]: Yeah I agree that this belongs in #77, as that is also where I introduced it, and I'll further comment on it within that PR. I simply also added this PR already as I thought they were 2 different decisions to make, and I guess I was on a roll and wanted to submit both ;)

**Comment by emilgoldsmith** [comment]
Yeah I'm actually also quite happy with the design "dream" anyway, which I guess is something like what it looks like in my environment at the moment.

Let's see if we can make that "dream" a reality on all platforms haha

**Comment by martynchamberlin** [comment]
Oh. I hadn't realized that this change was visually needed for this PR's changes, but now I see what you mean. Ok let's try to get the width changes removed for macOS but keep them for the other platforms (and keep the other changes for macOS) and we should be good.

**Comment by emilgoldsmith** [comment]
Yep, I think for now I'll focus on making #77 look good, and when we both agree that that's working for all platforms we can try implementing this again so that also looks good everywhere

**Comment by emilgoldsmith** [comment]
@martynchamberlin I just merged in my changes from #77 and made everything work together. Hope everything looks good on all platforms now! :)

**Comment by emilgoldsmith** [comment]
Does this also work equally good on Mac and Windows as #77 ? In that case if you can approve these changes I'll also merge this in :)

**Comment by emilgoldsmith** [comment]
Okay I merged in all changes from master to this branch and #77, I think merge went quite clean and of course ran the extension, don't think anything should change on Mac or Windows as I simply just merged the codes, feel free to check before you approve though.

But yeah, otherwise just approve the changes for these two PRs next time you're online, I'll merge them into master, and we'll have a pretty scrollbar all round!

**Pull request merged**